Initializes a standalone graphql app only responsible to fetch items from rethinkdb

pagidas opened the pull request:
To test
-------
Run `./spin_all.sh build` which wil force build all the containers in docker-compose.
(make sure you `chmod +x` to allow run permissions)

Then, if everything goes smoothly, and by hitting `localhost:8000` you can see messages coming into `messages` table in the rethinkdb. Go to `localhost:9090/graphiql` and try the following query:
```https
{
  messages(mostRecent: 3) {
    productId
    price
    time
  }
}
```
This will order all messages by time in descending, and will fetch 3 of them but pluck only `productId`, `price` and `time`. Of course, you can run the query without the `mostRecent` which will fetch everything. You can ask for whatever fields you want. As expected from the previous implementation of graphql we did in `coinbase-adapter`.

Graphiql is a web client which knows how to do graphql requests, and it's pointing to the service we're spinning. So instead of using postman, you can use that client provided with the app itself.

pagidas committed:
WIP Implements most recent fetch in graphql

pagidas committed:
Implements a hello graphql server

pagidas committed:
Minor improvements

pagidas committed:
WIP: Binds rehinkdb into graphql factory

pagidas committed:
WIP: Binds rethinkdb into graphql factory

pagidas committed:
Stores time messages in string, easier to fetch

pagidas committed:
Fetches coinbase messages using graphql

pagidas committed:
Implements get mostRecent messages flow in graphql

pagidas committed:
Inits graphql service in docker-compose

ngiachou commented:
Test failed.
```
Running all containers from previously built images...
Creating network "cryptobot_default" with the default driver
Building graphql
Step 1/11 : ARG VERSION=14
Step 2/11 : FROM openjdk:${VERSION}-alpine as BUILD
 ---> 8273876b08aa
Step 3/11 : COPY *.gradle gradle.* gradlew /src/
 ---> 2d4d4aa4bfc2
Step 4/11 : COPY gradle /src/gradle
 ---> 38b821b3798b
Step 5/11 : WORKDIR /src
 ---> Running in 16717ca09403
Removing intermediate container 16717ca09403
 ---> f52076b25ff7
Step 6/11 : RUN ./gradlew --version
 ---> Running in 729bc3d95f2a
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain
Caused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain
ERROR: Service 'graphql' failed to build: The command '/bin/sh -c ./gradlew --version' returned a non-zero code: 1
Done! All services running in the background
                          to see the containers run: $ docker ps -a
```

pagidas commented:
Try now -- it was missing the `gradle-wrapper.jar` because it was ignored in a local .gitignore file, which I removed, and added in the root one to _not_ ignore `.jar` files on gradle folders
